.eslintrc

```diff
@@ -13,6 +13,17 @@
   },
   "settings": {
     "import/resolver": {
+     "alias" : {
+        "map": [
+          ["helpers",  "./docs/src/helpers" ],
+          ["components", "./docs/src/components"],
+          ["static",  "docs/src/static" ],
+          ["backpack",  "./backpack" ],
```

Comment:
I thought we don't have this repo included in Backpack anymore as we use the npm packages directly?